src/modules/material-requests/controllers/material-requests.controller.ts

```diff
@@ -0,0 +1,39 @@
+import { Controller, Get, Query, UseGuards } from '@nestjs/common';
+import { ApiOperation, ApiTags } from '@nestjs/swagger';
+import { IPagination } from '@studiohyperdrive/pagination';
+
+import { MaterialRequestsQueryDto } from '../dto/material-requests.dto';
+import { MaterialRequest } from '../material-requests.types';
+import { MaterialRequestsService } from '../services/material-requests.service';
+
+import { SessionUserEntity } from '~modules/users/classes/session-user';
+import { Permission } from '~modules/users/types';
+import { RequireAllPermissions } from '~shared/decorators/require-permissions.decorator';
+import { SessionUser } from '~shared/decorators/user.decorator';
+import { LoggedInGuard } from '~shared/guards/logged-in.guard';
+
+@UseGuards(LoggedInGuard)
+@ApiTags('MaterialRequests')
+@Controller('material-requests')
+export class MaterialRequestsController {
+	constructor(private materialRequestsService: MaterialRequestsService) {}
+
+	@Get('personal')
+	@ApiOperation({
+		description: 'Get Material Requests endpoint for User.',
+	})
+	@RequireAllPermissions(
+		Permission.READ_PERSONAL_APPROVED_MATERIAL_REQUESTS,
+		Permission.MANAGE_ACCOUNT
```

Comment:
added permissions need to be documented in this confluence doc, so they can be added to the database during deploy:
https://meemoo.atlassian.net/wiki/spaces/HA2/pages/3770548456/Draaiboek+ARC+Release+1.4

Reply:
Adjusted and added to confluence doc